docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -98,7 +98,9 @@ The `sub` claim, which was always added to the access token, is now added by def
 
 The `Subject (sub)` mapper is configured by default in the `basic` client scope. Therefore, no extra configuration is required after upgrading to this version.
 
-Only in the case you are using `Pairwise subject identifier` mapper to map `sub` claim for access token you should disable or remove `Subject (sub)` mapper.
+Only in the case you are using `Pairwise subject identifier` mapper to map `sub` claim for access token you can consider disable or remove `Subject (sub)` mapper, however it is not strictly needed
+as the `Subject (sub)` protocol mapper is executed before `Pairwise subject identifier` mapper and hence the `pairwise` value will override the value added by `Subject (sub)` mapper.
+This may apply also to other custom protocol mapper implementations, which override `sub` claim, as `Subject (sub)` mapper is currently executed as first protocol mapper.
```

Comment:
```suggestion
This may apply also to other custom protocol mapper implementations, which override the `sub` claim, as the `Subject (sub)` mapper is currently executed as first protocol mapper.
```